interfaces/core.go

```diff
@@ -216,6 +216,22 @@ type ConfigfilesUser interface {
 	PathPatterns() []string
 }
 
+// SymlinksUser must be implemented by Interfaces that use the symlinks backend.
+type SymlinksUser interface {
+	// Directories is a list of directories that might contain symlinks
+	// under control of snapd. They are understood to be under control if
+	// they point to inside snap content or data. These directories apply
+	// to either the rootfs or to the mount namespace of a snap (TODO).
+	// AddSymlink from the backend is called to add a symlink that must be
+	// created. Symlinks that point to a snap that are not in the table are
+	// removed.
```

Comment:
I'm not sure I understand entirely what the last sentence is about, what does "table" refer to in there?